Lib/http/client.py

```diff
@@ -1050,6 +1050,8 @@ def _send_output(self, message_body=None, encode_chunked=False):
 
                 if encode_chunked and self._http_vsn == 11:
                     # chunked encoding
+                    if isinstance(chunk, str):
+                        chunk = chunk.encode('ascii')
```

Comment:
Why the ascii codec? This sounds like it would take us back to the dark ages of python 2, with text implicitly being changed into bytes using the worst possible choice of codecs that seems to work fine on an American programmer's tests and then raise UnicodeEncodeErrors when some user tries to use a non-ASCII character.

Reply:
because the line [1055](https://github.com/python/cpython/pull/11976/files#diff-b1e875fb060aa473c57a8e87bf903a76R1055) uses the ascii encoding.

Do you have an other idea?